deploy/helm/kubecf/assets/scripts/jobs/postgres/bpm_process.sh

```diff
@@ -12,6 +12,8 @@ wait_for_file() {
 # shellcheck disable=SC1091
 source /var/vcap/jobs/postgres/bin/pgconfig.sh
 
+chmod --recursive 0700 /var/vcap/store/postgres/
```

Comment:
Would it be possible to add a comment on _why_ we're doing this?

Reply:
added.